benchmarking/commons/hpo_main_common.py

```diff
@@ -115,6 +115,12 @@ def parse_args(
             "where A = n_workers and B = benchmark.n_workers"
         ),
     )
+    parser.add_argument(
+        "--use_long_tuner_name_prefix",
+        type=int,
+        default=1,
+        help="Use descriptive tuner name prefix for storing results?",
+    )
```

Comment:
This would be a breaking change. Do we really want to set this as the new default behavior? 

(Maybe we do, but we should weigh the cost for existing customers versus the ease-of-use benefits for future customers. Also, we're not yet at version 1.0.0, but once we say we're stable, we'll want to strive for backward compatibility.)

Reply:
I do not think this is really a breaking change. Results are stored in directories with prefix `tuner_name`. The only thing this PR is doing, is to use a longer value for `tuner_name`, when this is used with `benchmarking`.

I can set the default to 0, but the reason for this change is  pragmatic. Say you run experiments with several benchmarks, and suddenly you figure they need to be redone for one of the benchmarks. You need to remove the
result files in S3, but you cannot identify them from the directory names.

With this change, since benchmark name is part of the `tuner_name`, you can easily remove them using the S3 console. There is no drawback, except longer directory names.

Reply:
Spoke offline, makes sense, scope is also just limited to benchmarking